src/components/CloseButton/CloseButton.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './style.css';
+
+const CloseButton = ({ closeAction }) => (
+  <>
```

Comment:
you don't need fragment here

Reply:
Thank you for your note. I fixed it and it will in a new PR.